packages/docs/assets/colors.scss

```diff
@@ -0,0 +1,5 @@
+$active-blue: #2550C0;
```

Comment:
Can we do a couple of px of padding between buttons?
![image](https://user-images.githubusercontent.com/5394573/98268378-7ea4fb00-1f9d-11eb-9fa5-4f28df9f5312.png)